pkg/webhook/policy.go

```diff
@@ -253,6 +270,7 @@ func (h *validationHandler) getValidationMessages(res []*rtypes.Result, req *adm
 				logging.ConstraintAPIVersion, r.Constraint.GroupVersionKind().Version,
 				logging.ConstraintKind, r.Constraint.GetKind(),
 				logging.ConstraintAction, r.EnforcementAction,
+				logging.ConstraintViolations, r.Msg,
```

Comment:
constraint_violations is probably a misnomer here, as this is a single violation. constraint_violations is used in audit to list the one-or-more violations associated with a given constraint.

Should we modify `msg` to show the violation message?

Reply:
I thought this wasn't the right way to do add the msg/ use the constraint violations! Thanks for the explanation. 

I think adding it to `msg` is fine as long as we append it. While not a great idea, I know folks may construct brittle log scanners that would look for a string. So we probably do not take out the original string msg imo. Wdyt?

Reply:
Log scanners should look for the "process" and "event type" fields for stability (treating the log as json-formatted), as those fields have set meanings.

If we assume we want to avoid breaking brittle scanners, then we wouldn't be able to change any piece of the log message. For example, looking for the regex: `"denied admission"` would break under your proposal as much as changing the message entirely.

That being said, I'm not super interested in bikeshedding the contents of the message -- we should assume it's freely changeable in the future if we don't like the result.

Reply:
Backwards compatibility contract, for reference: https://github.com/open-policy-agent/gatekeeper/blob/master/docs/Backwards_Compatibility.md